### ISSUE TITLE
Password secrets not working fine

### ISSUE THREAD

**Issue by RohitR1** [HummingbirdTechGroup/helm-pypicloud#1]
Password secrets not working fine

https://github.com/HummingbirdTechGroup/helm-pypicloud/blob/master/templates/deployment.yaml#L28
It's using `ADMIN_PASSWORD` instead of ADMIN_PASSWORD

@viktorradnai 

Logs: 
```bash
pypicloud-5f477d855b-f2jp5 init-config ++ awk -F= '$1 ~ /PASSWORD/ { print $1 }'
pypicloud-5f477d855b-f2jp5 init-config ++ set
pypicloud-5f477d855b-f2jp5 init-config + for p in `set | awk -F= '$1 ~ /PASSWORD/ { print $1 }'`
pypicloud-5f477d855b-f2jp5 init-config + k=ADMIN_PASSWORD
pypicloud-5f477d855b-f2jp5 init-config + v=ADMIN_PASSWORD
pypicloud-5f477d855b-f2jp5 init-config ++ openssl passwd -stdin -6
pypicloud-5f477d855b-f2jp5 init-config ++ echo ADMIN_PASSWORD
pypicloud-5f477d855b-f2jp5 init-config + export 'ADMIN_PASSWORD=$6$ZnD.I7Tfj.an5C.W$LsB49bEzrVICdZinFsv4i9e/yz7quZK2owTE9USl8vw0GMq1J0FSLnx417H8s6cK5orTP2TUHQYFkMX/GZsS81'
pypicloud-5f477d855b-f2jp5 init-config + ADMIN_PASSWORD='$6$ZnD.I7Tfj.an5C.W$LsB49bEzrVICdZinFsv4i9e/yz7quZK2owTE9USl8vw0GMq1J0FSLnx417H8s6cK5orTP2TUHQYFkMX/GZsS81'
pypicloud-5f477d855b-f2jp5 init-config + for p in `set | awk -F= '$1 ~ /PASSWORD/ { print $1 }'`
pypicloud-5f477d855b-f2jp5 init-config + k=USER_PASSWORD
pypicloud-5f477d855b-f2jp5 init-config + v=USER_PASSWORD
pypicloud-5f477d855b-f2jp5 init-config ++ echo USER_PASSWORD
pypicloud-5f477d855b-f2jp5 init-config ++ openssl passwd -stdin -6
pypicloud-5f477d855b-f2jp5 init-config + export 'USER_PASSWORD=$6$xTOjZLuAFLjvDciD$vldXYE1jvjvBtzlCN/T4X2ju84kdFa.sHdcrveXMsA7eBUnXKfX1fmfOyaaTrcxa5Mh2Jq6iRAG9JDesiU/2c0'
pypicloud-5f477d855b-f2jp5 init-config + USER_PASSWORD='$6$xTOjZLuAFLjvDciD$vldXYE1jvjvBtzlCN/T4X2ju84kdFa.sHdcrveXMsA7eBUnXKfX1fmfOyaaTrcxa5Mh2Jq6iRAG9JDesiU/2c0'
```

**Comment by RohitR1** [comment]
There are few problem with envsubst like 
ValueError: hash could not be identified

```bash
envsubst to replace environment variables, it replaces "$rounds" with empty string, and it really confuses passlib
```

**Comment by viktorradnai** [comment]
Hi @RohitR1 , I have re-implemented the password crypting using Helm and removed the init container. Could you please test the latest master (0.4.0 version of the chart)? Note that it will only work with the `stevearc/pypicloud:latest` image and not `1.1.2` as the required change is not released yet.

**Comment by viktorradnai** [comment]
I'm going to close this as it should have been resolved with 0.4.0. Please let me know if this isn't the case.
